sn_registers/src/register.rs

```diff
@@ -170,12 +168,6 @@ impl Register {
         Ok(SignedRegister::new(self, signature))
     }
 
-    #[cfg(test)]
-    pub fn new_owned(owner: PublicKey, meta: XorName) -> Self {
-        let permissions = Default::default();
-        Self::new(owner, meta, permissions)
-    }
-
     /// Return the address.
     pub fn address(&self) -> &RegisterAddress {
```

Comment:
we could replace the function name with an overview of what the comment above is saying, removing the need for the comment and making the function name more descriptive